<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view::PlayersWindow</class>
 <widget class="QMainWindow" name="view::PlayersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Choose Colors</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="lblPlayer3">
         <property name="text">
          <string>Player 3:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblPlayer6">
         <property name="text">
          <string>Player 6:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer1" native="true">
         <property name="minimumSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer4" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer3" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblPlayer4">
         <property name="text">
          <string>Player 4:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtPlayer1">
         <property name="text">
          <string>Player 1</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblPlayer8">
         <property name="text">
          <string>Player 8:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtPlayer2">
         <property name="text">
          <string>Player 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPlayer1">
         <property name="text">
          <string>Player 1:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtPlayer5">
         <property name="text">
          <string>Player 5</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer7" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="txtPlayer8">
         <property name="text">
          <string>Player 8</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblPlayer7">
         <property name="text">
          <string>Player 7:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer6" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer2" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer5" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPlayer2">
         <property name="text">
          <string>Player 2:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtPlayer3">
         <property name="text">
          <string>Player 3</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtPlayer6">
         <property name="text">
          <string>Player 6</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="view::ColorPickerWidget" name="clrPlayer8" native="true">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: red;
border: 1px solid;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblPlayer5">
         <property name="text">
          <string>Player 5:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtPlayer4">
         <property name="text">
          <string>Player 4</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="txtPlayer7">
         <property name="text">
          <string>Player 7</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="chkFastStart">
         <property name="text">
          <string>Fast Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Start Game</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>view::ColorPickerWidget</class>
   <extends>QWidget</extends>
   <header>view/colorpickerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
